Cargo.toml

```diff
@@ -28,18 +28,18 @@ overflow-checks = true
 #backtraces = ["cosmwasm-std/backtraces"]
 
 [dependencies]
-cosmwasm-std = { version = "1.0.0-beta5" }
-cosmwasm-storage = { version = "1.0.0-beta5", features = ["iterator"] }
-cw-storage-plus = { version = "0.12.1" }
-provwasm-std = { version = "1.0.0-beta2" }
-rust_decimal = { version = "1.14" }
-schemars = "0.8.1"
-semver = "1.0.3"
-serde = { version = "1.0.127", default-features = false, features = ["derive"] }
-serde_json = "1.0.64"
-thiserror = { version = "1.0.23" }
-uuid = { version= "0.8.2" }
+cosmwasm-std = { version = "1.2.3" }
+cosmwasm-storage = { version = "1.2.3", features = ["iterator"] }
+cw-storage-plus = { version = "1.0.1" }
```

Comment:
It'll be interesting to see what effect this has, if any on gas costs

Reply:
i think our biggest gas cost is actually the fact we use `Bucket` for bid/ask orders which is `cosmwasm-storage`.

We would need to migrate to `HashMap` from `cw-storage-plus` which has a note about how it uses less gas in the docs